### PR TITLE
Update URL of Cozy logo inside emails

### PR DESCRIPTION
**Pull request by clochix**

### CODE REVIEW

**Comment by aenario** [comment]
(same comment than https://github.com/cozy/cozy-calendar/pull/438)

File: server/helpers/thumb.coffee [hunk 1]
```diff
@@ -11,6 +11,19 @@ whiteList = [
     'image/png'
 ]
 
+# Convert degree, minutes, secondes into position x and y
+gpsDegToDec = (pos, posRef) -> # String to int
+
+    split = pos.split( /(\d+)\/(\d+), (\d+)\/(\d+), (\d+)\/(\d+)/ )
```
Comment (nono) [inline server/helpers/thumb.coffee:17]: This line looks weird to me. Maybe s/split/match/ ?

File: server/helpers/thumb.coffee [hunk 2]
```diff
@@ -11,6 +11,19 @@ whiteList = [
...
+    unless split[6]
+        splitAlt = pos.split( /(\d+)\/(\d+)/ ) # altitude format
+        coord    = splitAlt[1] / splitAlt[2]
+    else
+        coord    = split[1] / split[2] + \
+                   (split[3] / split[4]) / 60 + \
+                   (split[5] / split[6])/3600 # lat and long format
+    ref = if (posRef == 'S' or posRef == 'W') then -1 else 1
+    return ref * coord
```
Comment (nono) [inline server/helpers/thumb.coffee:26]: Does coffeelint accept this method?

File: server/models/photo.coffee [hunk 3]
```diff
@@ -46,10 +46,12 @@ module.exports = class Photo extends cozydb.CozyModel
 
         else res = @getBinary 'raw', (err) ->
             console.log err if err?
-            callbabk() if err?
+            callback() if err?
 
         res.on 'ready', (stream) =>
             Helpers.readMetadata stream, (err, data) =>
+                console.log err if err?
+                callback() if err?
```
Comment (nono) [inline server/models/photo.coffee:54]: I'd have exected something like:

``` coffee
if err?
    console.log err
    callback err
else
    # ...
```

File: client/app/views/map.coffee [hunk 4]
```diff
@@ -101,13 +110,14 @@ module.exports = class MapView extends BaseView
...
+                    description = photo.get 'description'
                     element.append img
                     element.append button
                     unless photo.get('description')?
-                    then element.append $ "<quote>#{photo.get 'description' }</quote>"
+                    then element.append $ "<quote>#{description}</quote>"
                     img[0].onload = () ->
 
-                        setTimeout () =>
+                        setTimeout () ->
```
Comment (nono) [inline client/app/views/map.coffee:120]: I thought coffeelint say something against `()` before `->`.

File: client/app/views/map.coffee [hunk 5]
```diff
@@ -85,9 +94,9 @@ module.exports = class MapView extends BaseView
             gps      = photo.attributes.gps
             position = new L.LatLng gps.lat, gps.long
 
-            imgPath  = "photos/thumbs/#{photo.get('id')}.jpg"
-            text     = '<img src="images/spinner.svg" width="150" height="150"/>'
-            button   = '<button data-key="' + photo.get('id') +
+            imgPath = "photos/thumbs/#{photo.get('id')}.jpg"
+            text    = '<img src="images/spinner.svg" width="150" height="150"/>'
+            button  = '<button data-key="' + photo.get('id') +
```
Comment (nono) [inline client/app/views/map.coffee:99]: The `"""` could have been used here

**Comment by clochix** [comment]
@aenario I'll attach the logo to the message once cozy/cozy-clearance#27 has been published.

**Comment by aenario** [comment]
Could you check what's going on with travis ?

File: server/helpers/thumb.coffee [hunk 6]
```diff
@@ -11,6 +11,19 @@ whiteList = [
...
+gpsDegToDec = (pos, posRef) -> # String to int
+
+    split = pos.split( /(\d+)\/(\d+), (\d+)\/(\d+), (\d+)\/(\d+)/ )
+    unless split[6]
+        splitAlt = pos.split( /(\d+)\/(\d+)/ ) # altitude format
+        coord    = splitAlt[1] / splitAlt[2]
+    else
+        coord    = split[1] / split[2] + \
+                   (split[3] / split[4]) / 60 + \
+                   (split[5] / split[6])/3600 # lat and long format
```
Comment (nono) [inline server/helpers/thumb.coffee:24]: And I don't like:

``` coffee
unless cond
   # [1]
else
   # [2]
```

I prefer:

``` coffee
if cond
   # [2]
else
   # [1]
```

**Comment by clochix** [comment]
@aenario don't know what's happening to Travis, all tests are passing when I run them on my local computer :-S

**Comment by nono** [comment]
@clochix it looks like the DS has crashed. Can you add an `after-failure` section to travis, like https://github.com/cozy-labs/cozy-desktop/blob/master/.travis.yml#L59-L64 ?

**Comment by clochix** [comment]
Users are awaiting this fix for more than one week now, Travis is failing because of a DS problem.

**Comment by nono** [comment]
@clochix and I am awaiting for answers to my comments for more than one week too ;-)

**Comment by nono** [comment]
@clochix it has been merged, via #169

**Pull request closed without merging**